scripts/utils/migrate/instruction-set.js

```diff
@@ -388,13 +388,63 @@ module.exports = [
       icon: 'nr-fso',
     },
   },
-
-  // ----- New Relic only ----- \\
   {
     type: INSTRUCTIONS.UPDATE,
     path: ['New Relic only'],
     node: {
       rootNav: false,
     },
   },
+  {
+    type: INSTRUCTIONS.UPDATE,
+    path: ['Agents', 'PHP agent', 'PHP agent API'],
+    node: {
+      path: '/docs/agents/php-agent/php-agent-api',
+    },
+  },
+  {
+    type: INSTRUCTIONS.UPDATE,
+    path: ['Agents', '.NET agent', 'NET agent API'],
+    node: {
+      path: '/docs/agents/net-agent/net-agent-api',
+    },
+  },
+  {
+    type: INSTRUCTIONS.UPDATE,
+    path: ['Agents', 'Python agent', 'Python agent API'],
+    node: {
+      path: '/docs/agents/python-agent/python-agent-api',
+    },
+  },
+  {
+    type: INSTRUCTIONS.UPDATE,
+    path: ['Browser monitoring', 'Browser agent and SPA API'],
+    node: {
+      path: '/docs/browser/new-relic-browser/browser-agent-spa-api',
+    },
+  },
+  {
+    type: INSTRUCTIONS.UPDATE,
+    path: [
+      'Full-Stack Observability',
+      'Mobile monitoring',
+      'New Relic Mobile Android',
+      'Android SDK API',
+    ],
+    node: {
+      path: '/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api',
+    },
+  },
+  {
+    type: INSTRUCTIONS.UPDATE,
+    path: [
+      'Full-Stack Observability',
+      'Mobile monitoring',
+      'New Relic Mobile iOS',
+      'iOS SDK API',
+    ],
+    node: {
+      path: '/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api',
+    },
+  },
```

Comment:
I thought there were more agents (Ruby, Java, etc.). Could you make sure we have all of them in here?

Reply:
yep, those are the only agents that host api doc on docs site.
* apm agents: php, net, python
* browser agent
* mobile: ios, android

Reply:
Oh duh! For some reason I was thinking all of them had API docs. Glad you were working on this issue and not me!! 🤦